flocker/node/script.py

```diff
@@ -66,6 +68,40 @@ def flocker_dataset_agent_main():
     agent_script = AgentScript(service_factory=service_factory.get_service)
     options = DatasetAgentOptions()
 
+    # Use CPU time instead of wallclock time.
+    # The control service does a lot of waiting and we do not
```

Comment:
This was copied (somewhat in a rush) from `flocker/control/script.py` and was going to be included as part of [FLOC-3142](https://clusterhq.atlassian.net/browse/FLOC-3142). Do you think we should just tidy this up here or include it as part of a different PR?